src/App.js

```diff
@@ -2,20 +2,92 @@ import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
-class App extends Component {
+class App extends Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      page: 0
+    };
+  }
+  onPageChange() {
+    this.setState( {
+      page: this.state.page + 1
+    } );
+  }
+
   render() {
+    if (this.state.page === 0) {
```

Comment:
This is great for the first set of work, but the way this works here the page will keep incrementing but after the first screen it will only over show `Instructions`. 
 We can tackle that in the next pull request though.